### DISCUSSION TITLE
Need a way to import data from Last.fm via the website

### DISCUSSION THREAD

**Discussion by mattl** [foocorp/hacienda#92] [Future enhancements]
You should be able to enter your Last.fm username and we'll handle the rest.

**Comment by inan-hira** [comment]
Would this new method handle duplicates? I started using last.fm before libre.fm, but have been scrobbling to both. As a result, my last.fm account has more listens.

**Comment by mattl** [comment]
Yes it would

**Comment by alxtrnr** [comment]
May be worth having a look here https://github.com/digitalheir/lastfm-to-librefm-exporter I used it successfully some years ago. Currently the token that it asks Libre.fm to generate is not generated hence does not work right now.

**Comment by pluc** [comment]
Some more things

https://github.com/maxkueng/lastfmexport
https://github.com/nicolasm/lastfm-export
https://www.forceflow.be/2012/07/10/backing-up-last-fm-scrobbles/

We used to have a tool called LastToLibre? that lived here: http://bugs.foocorp.net/projects/librefm/wiki/LastToLibre - what happened to that?

**Comment by MitenkovIvan** [comment]
https://github.com/dbeley/lastfm-scraper
https://github.com/dbeley/lastfm-to-librefm <- still works well for importing last.fm scrobbles into libre.fm
